### DISCUSSION TITLE
T5 and T6 Trim Switches Do Not Work in Companion 2.10

### DISCUSSION THREAD

**Discussion by FrqFlyer** [EdgeTX/edgetx#5143] [General]
Trying to use T5 and T6 switches as toggle switches.  I use the following steps.

1.  Disable T5 and T6 Trim switches in Flight Mode 0.
2. Create a logical switch with Sticky, i,e., Sticky, T5+, T5-.  Call it L07
3. Add a mix channel to test  T5.  In the simulator in I cannot get L07.

On the Transmitter this logic works great.  This Youtube video illustrates it really well on the transmitter:  Manage your [#OpenTX](https://www.youtube.com/hashtag/opentx) trims like a PRO!.   https://www.youtube.com/watch?v=YuibLknMsx4

**Comment by pfeerick** [comment]
Can't reproduce this with Companion 2.10 Simulator (either when created in Companion first, or configured entirely in Simulator)... are you looking down the source list far enough?
![image](https://github.com/EdgeTX/edgetx/assets/5500713/393ee7f1-09fa-4eee-8271-53772ec0da31)
![image](https://github.com/EdgeTX/edgetx/assets/5500713/03284556-145f-4ed2-8beb-2cb8a671091b)
![image](https://github.com/EdgeTX/edgetx/assets/5500713/180aa2a4-5f97-421f-9de7-af185ba67903)
![image](https://github.com/EdgeTX/edgetx/assets/5500713/e059382a-f2b5-4c9e-81fd-14ad1ec9aae3)
![image](https://github.com/EdgeTX/edgetx/assets/5500713/9912cc95-4f05-42d4-92a0-66c287d2b8e1)

**Comment by FrqFlyer** [comment]
Thank you for your quick response.  I appreciate it.

I cannot make it work in Companion 2.10.  However, it works great on the
transmitter.  I'm using it for thrust reverse on the eFlite A10 -- works
great on the plane.

I even set it up on the transmitter.  I made sure it worked correctly on
the plane.  Then I read that model setup into Companion.  Still doesn't
work.  Looks like a bug or something the developers haven't worked on yet.

Regards,
Gary Armstrong

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Reply by pfeerick** [reply]
Can you share one of the models you are trying to get working - you will need to upload it on github, not via email attachment (look for the attach paperclip just above the input textbox) - as all those screenshots were *from* Companion 2.10... so clearly it's not a simple *just doesn't work at all* bug :rofl: It is entirely possible that Companion isn't reading the settings in properly.

**Reply by pfeerick** [reply]
I.e. if I just try from scratch in companion, and check it in simulator (not taking into account L07, etc this time), this works:
https://github.com/EdgeTX/edgetx/assets/5500713/e3f1c613-1ddd-45be-a3bd-b39be5b79196

**Comment by FrqFlyer** [comment]
Peter, thanks for your assistance.

I my setup for my eflite A10 and the T5 switch works great. No issues at
all.

However when I connect my transmitter to Companion and load the models into
Companion, the T5 simply doesn’t work.

I just wanted the developers to know that there is a possible bug in
Companion.

The link I provided to FPV University video is exactly what I’m using.

Again, on the TX and the model, it works perfectly.

Regards,
Gary
